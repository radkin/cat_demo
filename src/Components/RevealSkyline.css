.photo {
  width: 2050px;
  height: 320px;
  padding-top: 40px;
}

.skyBack {
  background-color: white;
}
