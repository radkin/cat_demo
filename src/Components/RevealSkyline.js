import React from 'react';
import Zoom from 'react-reveal/Zoom';
import rrConfig from 'react-reveal/globals';

import './RevealSkyline.css';

rrConfig({ ssrFadeout: true });

class ZoomExample extends React.Component {
  render() {
    return (
      <div className="skyBack">
        <Zoom>
          <img src={require('../assets/images/header.jpg')} alt="city skyline" className="photo"/>
        </Zoom>
      </div>
    );
  }
}

export default ZoomExample;
