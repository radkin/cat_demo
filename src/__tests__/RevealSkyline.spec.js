import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import RevealSkyline, { Counter, dataReducer } from '../Components/RevealSkyline';

describe('RevealSkyline', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<RevealSkyline />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
