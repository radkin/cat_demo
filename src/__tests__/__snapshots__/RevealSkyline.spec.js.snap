// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RevealSkyline snapshot renders 1`] = `
<div
  className="skyBack"
>
  <img
    alt="city skyline"
    className="react-reveal photo"
    src="header.jpg"
    style={
      Object {
        "opacity": undefined,
      }
    }
  />
</div>
`;
